language/stdlib/compiled/src/lib.rs

```diff
@@ -36,23 +36,19 @@ pub const ERROR_DESCRIPTIONS: &[u8] =
 // specified either by the MOVE_NO_USE_COMPILED env var, or by passing the "StdLibOptions::Fresh"
 // option to `stdlib_modules`.
 static COMPILED_MOVELANG_STDLIB: Lazy<Vec<CompiledModule>> = Lazy::new(|| {
-    let mut modules: Vec<(String, CompiledModule)> = COMPILED_STDLIB_DIR
+    let mut modules: Vec<(&str, CompiledModule)> = COMPILED_STDLIB_DIR
```

Comment:
Could you collect this into a `BTreeMap` and then no sorting is needed later on? i.e.
```rust
let modules: BTreeMap<&str, CompiledModule> = <same code you have here>;
for (_, module) in modules.into_iter() { .... }
...
```